src/main/java/com/networknt/schema/TypeFactory.java

```diff
@@ -75,6 +75,11 @@ public static JsonType getSchemaNodeType(JsonNode node) {
      * @return the json type
      */
     public static JsonType getValueNodeType(JsonNode node, SchemaValidatorsConfig config) {
+        if (node == null) {
+            // This returns JsonType.UNKNOWN to be consistent with the behavior when

```

Comment:
```suggestion
            // JsonType.UNKNOWN is consistent with JsonNodeType.MISSING
```
Trying to make it a bit concise?

Reply:
Resolved without any update, somehow?

Reply:
Thanks for the suggestion. However your suggestion doesn't convey the intent of the comment that the return result should match the result in the later switch statement for missing node.

Reply:
Ah I see, thanks for your response 👍🏻 .